medicaltorch/transforms.py

```diff
@@ -432,13 +451,13 @@ def __call__(self, sample):
         input_data = sample['input']
         gt_data = sample['gt'] if self.labeled else None
         if np.random.randint(2) == 1:
-            input_data = np.flip(input_data, axis=0).copy()
+            input_data = np.flip(input_data, axis=1).copy()
```

Comment:
axis=1 since axis=0 represents the channels, okay, all good.

Reply:
Yup! Looking if I can do that more often instead of using `for`  loop.